### PR TITLE
[jit] add 'all' builtin

### PR DESCRIPTION
**Pull request by ghost**
[jit] add 'all' builtin

### CODE REVIEW

File: torch/csrc/jit/register_prim_ops.cpp
Comment (driazati) [inline torch/csrc/jit/register_prim_ops.cpp]: There is already an op for `aten::all` in the ops that are generated from the code in `aten`, so I don't think we need this op, just adding it to the builtins list should be enough.

**Comment by michaelarg** [comment]
@driazati I have removed the ops and left in the builtin line. Is this what you were after? Thanks

**Comment by ezyang** [comment]
Could you please cleanup the spurious whitespace changes in this patch, and add a test? I don't know the code so I don't know if the change is right, but it looks reasonable.

File: torch/csrc/jit/register_prim_ops.cpp
Comment (driazati) [inline torch/csrc/jit/register_prim_ops.cpp]: There's already an operator for `aten::all` that's generated from an ATen operator (`tensor.all()`), you can find it in `register_generated_aten_ops_1.cpp` after you build the code. For that reason I don't think we need any code here for this op, just adding it to the list of globals in `compiler.cpp` should be enough.

**Review by driazati — COMMENTED** [review]
This is good to go if you can add back in the test you had earlier and clean up the whitespace changes

File: torch/csrc/jit/register_prim_ops.cpp
Comment (michaelarg) [reply torch/csrc/jit/register_prim_ops.cpp]: Sorry I'm such a noob but hopefully this looks ok. If not can you give me a little bit more detail on the whitespace cleanup. Thanks

**Review by driazati — APPROVED** [review]
Looks good, the merge conflict looks easy to fix; could you either check the "Allow edits from maintainers" checkbox on the GitHub sidebar or apply this patch yourself to remove the trailing whitespace (our linters will complain)


```diff
diff --git a/test/test_jit.py b/test/test_jit.py
index 4e8e55804..b36db1870 100644
--- a/test/test_jit.py
+++ b/test/test_jit.py
@@ -6288,9 +6288,9 @@ a")
         m = M()
         self.assertEqual(m(), 10)
 
-    def test_script_module_for2(self):  
+    def test_script_module_for2(self):
         class Sub(torch.jit.ScriptModule):
-            def __init__(self):            
+            def __init__(self):
                 super(Sub, self).__init__(False)
                 self.weight = nn.Parameter(torch.randn(2))
 
diff --git a/torch/csrc/jit/register_prim_ops.cpp b/torch/csrc/jit/register_prim_ops.cpp
index e0ca021ec..27393d2ea 100644
--- a/torch/csrc/jit/register_prim_ops.cpp
+++ b/torch/csrc/jit/register_prim_ops.cpp
@@ -1842,7 +1842,7 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__and__, a& b),
     DEFINE_INT_OP(aten::__or__, a | b),
     DEFINE_INT_OP(aten::__xor__, a ^ b),
-    
+
     Operator(
         "prim::abs(int x) -> int",
         [](Stack& stack) {
```

**Comment by michaelarg** [comment]
@driazati I have allowed edits from maintainers. Thank you for helping me work through my first issue!

**Pull request closed without merging**
